Makefile

```diff
@@ -1,6 +1,6 @@
 .ONESHELL:
 
-bench: CFLAGS = $(shell pkg-config --cflags libpmemkv) -DOS_LINUX -fno-builtin-memcmp -march=native -DNDEBUG -O2 -std=c++11
+bench: CFLAGS = $(shell pkg-config --cflags libpmemkv) -DOS_LINUX -fno-builtin-memcmp -march=native -DNDEBUG -O3 -std=c++11
```

Comment:
Just increasing O2 to O3 is not always the best idea - the performance might be better on O2 for some cases. If you have some performance numbers to support this change that's fine but if not I'd prefer to leave it as was.